_posts/en/newsletters/2020-04-08-newsletter.md

```diff
@@ -107,13 +105,14 @@ Later in the meeting, the `TxDownloadState` data structure was discussed:
 - <details><summary>Describe the role of the <code>TxDownloadState</code> struct?</summary>
     A per-peer state machine, with timers, to coordinate requesting transactions
     from peers.</details>
-- <details><summary>What are 3 <code>TxDownloadState</code> states a transaction can be in?</summary>
-  (1) announced, but not requested; (2) announced, requested, and waiting for
-  delivery; (3) announced, requested, but timed out.</details>
+- <details><summary>How could we improve <code>TxDownloadState</code> to
+  make it less likely to introduce tx relay bugs in future?</summary>
```

Comment:
suggestion: s/tx relay bugs in future/transaction relay bugs in the future/

Reply:
Done